dirty-lib.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/actcat/sideci_go_sandbox/lib"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/naoina/denco"
+	_ "github.com/naoina/kocha"
+	_ "github.com/revel/revel"
+	_ "github.com/zenazn/goji"
+)
+
+func HttpHandler(w http.ResponseWriter, r *http.Request, p denco.Params) {
+	foo := p.Get("foo")
+	w.Write([]byte(foo))
+}
+
+func UseLibFoo() {
```

Comment:
__[golint]__
* [golint] exported function UseLibFoo should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->